galsim/__init__.py

```diff
@@ -111,7 +111,7 @@
 
 # GSObject
 from .base import GSParams, GSObject, Gaussian, Moffat, Airy, Kolmogorov, Pixel, Box, TopHat
-from .base import Exponential, Sersic, DeVaucouleurs, Spergel
+from .base import Exponential, Sersic, DeVaucouleurs, Spergel, InclinedExponential
```

Comment:
This should probably get its own file, rather than add another class to base.  This isn't really a profile that we will consider a "base" class in GalSim, so better to have a new file for it.  (Spergel probably also ought to have gotten its own file as well for the same reason...)